Compile error ESP8266 Arduino IDE with example

Hi, I tried example with ESP8266 in Arduino IDE, I get this error:
```
In file included from C:\Users\j.jirutka\Documents\Arduino\libraries\IoTController\src/IoTController.h:7,
                 from C:\Users\j.jirutka\Documents\Arduino\libraries\IoTController\examples\RGB-Strip\RGB-Strip.ino:1:
C:\Users\j.jirutka\Documents\Arduino\libraries\IoTController\src/RedController.h:5:10: fatal error: ArduinoJson.h: No such file or directory
    5 | #include <ArduinoJson.h>
      |          ^~~~~~~~~~~~~~~
compilation terminated.
exit status 1
Nastala chyba při kompilaci u desky Generic ESP8266 Module.
```
After installing all missing libraries, I get this error:
```
C:\Users\j.jirutka\Documents\Arduino\libraries\IoTController\src\MqttController.cpp: In member function 'boolean MqttController::reconnect_mqtt()':
C:\Users\j.jirutka\Documents\Arduino\libraries\IoTController\src\MqttController.cpp:73:1: error: no return statement in function returning non-void [-Werror=return-type]
   73 | }
      | ^
cc1plus.exe: some warnings being treated as errors
exit status 1
Nastala chyba při kompilaci u desky Generic ESP8266 Module.
```

--- a/src/MqttController.cpp
+++ b/src/MqttController.cpp
@@ -61,13 +61,12 @@ boolean MqttController::isMqttConnected() {
   return isConnect;
 }
 
-boolean MqttController::reconnect_mqtt()
+void MqttController::reconnect_mqtt()
 {
   mqtt_client.disconnect();
   delay(100);
   PubSubClient mqtt_client(wifiClient);
   setup_mqtt();
-  return true;
 }
 
 boolean MqttController::send_mqtt_notification(String payload)

--- a/src/MqttController.h
+++ b/src/MqttController.h
@@ -37,7 +37,7 @@ class MqttController {
     void init(String serverName, String deviceType, String deviceId, String username, String password);
     void setup_mqtt();
     boolean isMqttConnected();
-    boolean reconnect_mqtt();
+    void reconnect_mqtt();
     boolean send_mqtt_notification(String payload);
     boolean send_mqtt_available();
     void mqttLoop();

